app/contexts/GlobalState.tsx

```
@@ -524,6 +525,17 @@ const GlobalStateProviderInner: React.FC<GlobalStateProviderProps> = ({
       entitlementApiResolvedUserId === user?.id) &&
     !entitlementRefreshRequested &&
     !automaticEntitlementRefreshPending;
+  const entitlementRefreshFailure = entitlementRefreshFailureRef.current;
```

ESLint

[error] Error: Cannot access refs during render

React refs are values that are not needed for rendering. Refs should only be accessed outside of render, such as in event handlers or effects. Accessing a ref value (the `current` property) during render can cause your component not to update as expected (https://react.dev/reference/react/useRef).

/home/jailuser/git/app/contexts/GlobalState.tsx:528:37
  526 |     !entitlementRefreshRequested &&
  527 |     !automaticEntitlementRefreshPending;
> 528 |   const entitlementRefreshFailure = entitlementRefreshFailureRef.current;
      |                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Cannot access ref value during render
  529 |   const tokenFreeAutomaticRefreshExhausted =
  530 |     subscriptionFromEntitlements === "free" &&
  531 |     entitlementRefreshFailure?.userId === user?.id &&

(react-hooks/refs)